client/payment-methods/index.js

```diff
@@ -145,10 +141,30 @@ const PaymentMethods = () => {
 	} = useContext( WCPaySettingsContext );
 
 	const { isUpeEnabled, status } = useContext( WcPayUpeContext );
+	const [ useIsSurveyModalOpen, setIsSurveyModalOpen ] = useState( false );
+	const [ useIsDisableModalOpen, setIsDisableModalOpen ] = useState( false );
 
 	return (
 		<>
-			{ 'error' === status && <UpeDisableError /> }
+			{ useIsDisableModalOpen ? (
+				<DisableUPEModal
+					enabledMethods={ enabledMethods }
+					setIsModalOpen={ setIsDisableModalOpen }
+					triggerAfterDisable={ () => setIsSurveyModalOpen( true ) }
```

Comment:
(minor) modals have an `onRequestClose` prop - what do you think about renaming `triggerAfterDisable` to `onRequestClose` in this component?